packages/client/hmi-client/src/components/widgets/tera-input-field.vue

```diff
@@ -4,18 +4,20 @@
 		<input
 			v-bind="attrs"
 			ref="inputField"
-			:value="modelValue"
+			:value="getValue()"
 			@input="updateValue"
 			:style="{ 'text-align': textAlign }"
+			@blur="unmask"
```

Comment:
Don't have a good place to put this, but there are two things worth mentioning related to how the template is used:
1. Note `input[type=number]` automatically validates the input is actually a number, which type=sci doesn't do
2. There is not such thing as `input[type=sci]`, it isn't proper HTML, if we are rolling our own stuff it is probably `input[type=text]`

Reply:
I needed a unique way to identify that we are using a custom scientific number input I thought that this might be the easiest way for us to use this in other components, using text would be just simply for text fields. 

The issue with using number is that using that type will try to auto parse into a number but we needed a custom string (the number with the spaces).  As far as I know using just number cannot do this.  i.e if i typed in `12345`.  The mask would change the value to `12 345` but the number type would not recognize this as invalid.